backend/LeetWars.Core/LeetWars.Core.BLL/Interfaces/IStarService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using LeetWars.Core.Common.DTO.ChallengeStar;
+
+namespace LeetWars.Core.BLL.Interfaces
+{
+    public interface IStarService
+    {
+        Task<ChallengeStarDto> Update(ChallengeStarDto challengeStarDto);
```

Comment:
do we really need a separate service for this?

Reply:
I just thought that we update stars not challenges, but now, after refactoring, it really looks like we change challenge, so we don't need starservice anymore